### PR TITLE
Update UpgradePage.tsx

### PR DESCRIPTION
**Pull request by oddlittlebird**
Edited UI text on the OSS Upgrade to Enterprise page. Mostly fixed capitalization errors.

### CODE REVIEW

**Review by osg-grafana — APPROVED** [review]

File: public/app/features/admin/UpgradePage.tsx [hunk 1]
```diff
@@ -125,8 +125,8 @@ const FeatureListing: React.FC = () => {
     <List>
       <Item title="Data source permissions" />
       <Item title="Reporting" />
-      <Item title="SAML Authentication" />
-      <Item title="Enhanced LDAP Integration" />
+      <Item title="SAML authentication" />
+      <Item title="Enhanced LDAP integration" />
       <Item title="Team Sync">LDAP, GitHub OAuth, Auth Proxy, Okta</Item>
       <Item title="White labeling" />
```
Comment (osg-grafana) [inline public/app/features/admin/UpgradePage.tsx:131]: Any chance this can be called "Rebranding"?

Comment (oddlittlebird) [reply public/app/features/admin/UpgradePage.tsx:131]: Probably not, but that is a @mjseaman question.

Comment (mjseaman) [reply public/app/features/admin/UpgradePage.tsx:131]: Yes good call out! I would like to call this feature "Custom branding." Open to debate on the name :) 

I think we should change it everywhere at once, and we'll have the opportunity to do that when we release some new functionality in an upcoming quarter. The new functionality would include custom UI colors and themes, and a user interface for uploading and managing custom graphics (as opposed to the .ini config options we have now). In the meantime I'd keep it as is.

**Review by mjseaman — APPROVED** [review]

**Review by mckn — APPROVED** [review]
LGTM!

I have run our tooling against the files to make this build pass. If you want to change texts in code in the future I strongly recommend to setup the environment so you get the tooling in place to prevent build failures.

**Pull request merged**